products/views.py

```diff
@@ -1,3 +1,24 @@
-from django.shortcuts import render
+import my_settings
 
-# Create your views here.
+from django.http                import JsonResponse
+from django.views               import View
+
+from products.models            import MainCategory, SubCategory
```

Comment:
```suggestion
from django.http     import JsonResponse
from django.views    import View

from products.models import MainCategory, SubCategory
```
```suggestion
from django.http                import JsonResponse
from django.views               import View

from products.models            import MainCategory, SubCategory
```
가장 긴 것을 기준으로 한 칸정도만 띄어주면 됩니다~

Reply:
넵 수정하겠습니다!